apps/example-ssr/src/pages/posts/[slug].astro

```diff
@@ -3,6 +3,8 @@ import Layout from "../../layouts/Layout.astro";
 import { useSanityClient } from "@sanity/astro";
 import PortableText from "../../components/PortableText.astro";
 import SanityImage from "../../components/SanityImage.astro";
+// import '@astrojs/prism'
+// import 'prismjs/components/prism-markup-templating'
```

Comment:
remove these?

Reply:
Yes indeed, sorry -- those were leftovers from chasing an apparent tree-shaking problem in recent Astro w/Netlify....